### ISSUE TITLE
[ Issue ] Kafka configuration update is not working as expected 

### ISSUE THREAD

**Issue by vmugutrao** [aliyun/terraform-provider-alicloud#3947]
### Terraform Version
Terraform v0.13.5

### Affected Resource(s)
- alicloud_alikafka_instance

### Terraform Configuration Files
```hcl
resource "alicloud_alikafka_instance" "devkafkainstance" {
  name            = "mytestinstance"
  spec_type       = "normal"
  deploy_type     = "4"
  eip_max         = "5"
  io_max          = "20"
  disk_type       = "1"
  disk_size       = "500"
  paid_type       = "PostPaid"
  service_version = "2.2.0"
  config          = jsonencode({ "kafka.log.retention.hours" = "72" }) 
  topic_quota     = "50"
  vswitch_id      = "vsw-h86lwdddmc36isszbpex7"
}

```

### Expected Behavior
By default when we create a kafka instance via terraform it set "Maximum Message Size" as 1MB.
When I am trying to set that to 10 MB, by specifying "kafka.message.max.bytes" config in instance config it plan as to replace whole instance.
When I changing this config via console, restart cluster, however my instance get retained (Config like endpoint, whitelisting and SASL users)

### Actual Behavior
It should not replace whole instance, rather it just update message size of instance, all configuration like endpoint, whitelisting and SASL users should retained.

### Steps to Reproduce
1. Create kafka instance with default config. 
2. Once instance is ready, change "kafka.message.max.bytes" config to "10485760" (10MB).
3. terraform plan (You should see instance will get replacement action)

**Comment by saeedgenmills** [comment]
Agree. Updating GUI retains all settings rather then re-creating everything.

**Comment by vmugutrao** [comment]
@xiaozhu36 @jameslcj  Can you help on this, I am stuck because of this ?

**Comment by tiankai66** [comment]
Your feedback has been received. We will process it as soon as possible.
Thank you for your trust and support of Alibaba Cloud!

**Comment by gmi-vishal** [comment]
Any update ??

**Comment by MrWolong** [comment]
According to your description, I ran it once, it's ok. pls try again.

**Comment by xuzhang3** [comment]
Closing this issue. Feel free to open a new issue if you still have questions.
